workspace/resource_notebook.go

```diff
@@ -272,6 +279,10 @@ func ResourceNotebook() common.Resource {
 				Path:      path,
 				Overwrite: true,
 			}
+			if createNotebook.Format == "" && createNotebook.Language != "" {
```

Comment:
Can users specify format? It's not documented on the public docs: https://registry.terraform.io/providers/databricks/databricks/1.61.0/docs/resources/notebook. If not, we can remove the first branch of the `if` since it is always true.